testing/tests/DevExpress.ui.widgets.editors/htmlEditorParts/tableResizingIntegration.tests.js

```diff
@@ -800,7 +899,7 @@ module('Table resizing integration', {
 
             const $highlightedElement = this.$element.find(`.${DX_HIGHLIGHTED_ROW_CLASS}`);
 
-            assert.roughEqual(parseInt($highlightedElement.css('top')) + DRAGGABLE_ELEMENT_OFFSET, $table.outerHeight(), 3.01);
+            assert.roughEqual(parseInt($highlightedElement.css('top')) + DRAGGABLE_ELEMENT_OFFSET, $table.outerHeight() - TABLE_BORDERS, 3);
```

Comment:
should it be `3.01` instead of `3`?

Reply:
no, it is not actually with TABLE_BORDERS